projects/1.19/assets/varied-adventure/va/lang/zh_cn.json

```
@@ -0,0 +1,251 @@
+{
+  "advancements.bleed.descr": "受到诅咒伤口的负面效果。",
+  "advancements.bleed.title": "流血！",
+  "advancements.call_a_monster.descr": "获得一个蛇笛。",
```

```suggestion
  "advancements.call_a_monster.descr": "获得一支蛇笛。",
```
`一根`之类的应该也行
善用量词，此类问题请全文通查